Genieo.adf

```diff
@@ -76,6 +76,7 @@
 		<File cruft="true">/Users/*/Library/Application Support/JakeCares</File>
 		<File cruft="true">/Users/*/Library/Application Support/Javeview</File>
 		<File cruft="true">/Users/*/Library/Application Support/Jimbrie</File>
+		<File cruft="true">/Users/*/Library/Application Support/Keka</File>
 		<File cruft="true">/Users/*/Library/Application Support/Kuklorest</File>
```

Comment:
I've used [Keka](http://www.kekaosx.com/en/) for a very long time. Are you certain this is malware?